elasticsearch/src/main/java/org/hibernate/search/elasticsearch/cfg/IndexSchemaManagementStrategy.java

```diff
@@ -14,10 +14,19 @@
 public enum IndexSchemaManagementStrategy {
 
 	/**
-	 * Indexes will never be created or deleted. The index schema is not managed by Hibernate Search.
+	 * Indexes will never be created or deleted. The index schema is not managed by Hibernate Search and is never checked.
```

Comment:
I'd simply say "is not checked" rather than "is never checked". Even better "is not validated".

The "never" is confusing as it suggests that we do some kind of operation continuously in time.